features/google/src/model/importers/utils/lang-constants.js

```diff
@@ -1,23 +1,29 @@
 export const langConstants = {
     AccessLogImporter: {
-        regex: /\/(Access Log Activity|Zugriffsprotokollaktivitäten|Aktiviteter i adgangsloggen)\/(Activities [-–] A list of Google services accessed by|Aktivitäten [-–] eine Liste der Google-Dienste, auf d|Aktiviteter [-–] en liste over Google-tjenester, som d)\.csv$/u,
+        regex: /\/(Access Log Activity|Zugriffsprotokollaktivitäten|Aktiviteter i adgangsloggen)\/(Activities [-–] A list of Google services accessed by|Aktivitäten [-–] eine Liste der Google-Dienste, auf d|Aktiviteter [-–] en liste over Google-tjenester, som d)\.csv$/,
```

Comment:
AFAIK, the `/u` mainly takes care of matching symbols as individual characters correctly - so it does seem like an unrelated (but still potentially helpful) thing.

Reply:
🤔 You have a point. I will test it.

Reply:
All tests fail when I add `u` at the end.

Reply:
That's... odd. But I'll accept it :D